setup.py

```diff
@@ -38,7 +38,7 @@ def get_static_files(path):
 
 install_requires = [
     'tornado>=4.0,<5',
-    'python-daemon<3.0',
+    'python-daemon<3.0'
```

Comment:
In the future, do not remove these trailing commas as they have the purpose of making append-diffs easier to read. :)